src/test_fixture/network/receive.rs

```diff
@@ -0,0 +1,57 @@
+use crate::test_fixture::network::stream::{StreamCollection, StreamKey};
```

Comment:
no functional changes here, just moving things around. In memory transport module was too big already